fab/inventory/icds

```diff
@@ -117,5 +120,20 @@ kafka0
 [zabbix:children]
 kafka0
 
+[patch:children]
+patch0
+
+[patch_clients:children]
+pg0
+pg1
+web0
+web1
+es0
+es1
+couch0
+celery0
+pil0
+kafka0
```

Comment:
is this 'all server except patch0'? If so I think we could generate the list dynamically. Maybe not worth it if it's hard though.